src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -946,6 +946,70 @@ RppStatus rppt_slice_host(RppPtr_t srcPtr,
                                 layoutParams,
                                 rpp::deref(rppHandle));
     }
+    return RPP_SUCCESS;
+}
+
+/******************** transpose_voxel ********************/
+
+RppStatus rppt_transpose_generic_host(RppPtr_t srcPtr,
+                                      RpptGenericDescPtr srcGenericDescPtr,
+                                      RppPtr_t dstPtr,
+                                      RpptGenericDescPtr dstGenericDescPtr,
+                                      Rpp32u *permTensor,
+                                      Rpp32u *roiTensor,
+                                      rppHandle_t rppHandle)
+{
+    RppLayoutParams layoutParams;
+    if ((srcGenericDescPtr->layout == RpptLayout::NCDHW) && (dstGenericDescPtr->layout == RpptLayout::NCDHW))
```

Comment:
Are these NCDHW/NDHWC still valid if the user is giving an nd transpose that uses your generic version?

Reply:
Removed them